notebooks/fireredtts2/fireredtts2.ipynb

```diff
@@ -0,0 +1,1419 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        "[S1]I think we should just talk about what happened and move on because there's going to be other jousts and Sir Saif isn't done yet. It's not, he's not, it's not done yet.",
</pre><p>I think it would be nice to add some extra description about what format it should be. What I mean is that should only start with [S1], [S2], [S3] or [S4].</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3120/discussion/#comment-2606921797'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>